foodsaving/invitations/serializers.py

```diff
@@ -45,3 +47,20 @@ class InvitationAcceptSerializer(serializers.Serializer):
     def update(self, invitation, validated_data):
         invitation.accept(self.context['request'].user)
         return invitation
+
+
+class InvitationResendEmailSerializer(serializers.Serializer):
+    from datetime import timedelta
+
+    created_at = serializers.DateTimeField()
+
+    def validate(self, attrs):
+        if not timezone.now() >= attrs['created_at'] + self.timedelta(hours=1):
```

Comment:
I'd move this into a permission class and specify it in the action as
```
permission_classes = (IsAuthenticated, CanResend)
```

There are some examples in `pickups/api.py`.

Reply:
Alright, I saw the examples, and I'll pick this up!